src/store/messages/saga.ts

```diff
@@ -362,6 +367,7 @@ export function* receiveNewMessage(action) {
       })
     ),
     call(sendBrowserNotification, channelId, message),
+    call(markAllMessagesAsRead, channelId, message.sender?.userId),
```

Comment:
I don't think this is right. We only want to mark the messages read if it's the conversation/channel that's currently open.

Reply:
Ah, this is also where we're using the wrong userId. Maybe just don't send the userId and let the other saga fetch the current user.